src/api/public/apidocs-new/components/schemas/request/request.yaml

```diff
@@ -79,95 +79,53 @@ properties:
   review:
     type: array
     items:
-      anyOf:
```

Comment:
@eduardoj something seems wrong now, the history elements are inside `review` as well now
![image](https://user-images.githubusercontent.com/22001671/186191473-651765a9-bb8b-4954-917b-71bd1671f294.png)

Reply:
Yes. There are `history` elements at both levels: `request` and `review`. In my development environment, for example:
```
> osc -A dosc api "/search/request?match=@id=2&withhistory=1"
<collection matches="1">
  <request id="2" creator="Iggy">
    <action type="submit">
      <source project="openSUSE:Leap:15.0" package="apache2"/>
      <target project="openSUSE:Factory" package="inreview"/>
    </action>
    <state name="review" who="Iggy" when="2022-08-22T08:06:58">
      <comment/>
    </state>
    <review state="new" when="2022-08-22T08:06:58" by_package="apache2" by_project="openSUSE:Leap:15.0"/>
    <review state="new" when="2022-08-22T08:06:58" by_user="repo-checker"/>
    <review state="accepted" when="2022-08-22T08:06:58" who="Admin" by_group="group_1">
      <comment/>
      <history who="Admin" when="2022-08-22T08:06:58">
        <description>Review got accepted</description>
      </history>
    </review>
    <review state="new" when="2022-08-22T08:06:58" by_group="review-team"/>
    <review state="new" when="2022-08-22T08:06:58" by_project="openSUSE:Factory:Staging:A"/>
    <history who="Iggy" when="2022-08-22T08:06:58">
      <description>Request created</description>
      <comment>Praesentium in nobis. Tempore omnis error. Ut voluptas quibusdam.</comment>
    </history>
    <description>Praesentium in nobis. Tempore omnis error. Ut voluptas quibusdam.</description>
  </request>
</collection>
```

Reply:
okay :)